factory/reader.js

```diff
@@ -24,38 +24,34 @@ function parseContent(html) {
       content = getTextFromHtml(articleMarkup);
       
       //@dfasoro added this.
-      var comments = [];
+      var comments = [], authors = [], body = [], time = [];
       var commentsNode = $('#comments');
       comments.push($('H4', commentsNode).first().text());
       
-      var authors = [], body = [], time = [];
-      
       try {
-          $('#comments-block DT.comment-author', commentsNode).each(function () {
-              var author_line = $(this).text().trim();
-              author_line = author_line.substr(0, author_line.length - 7).trim();
-              authors.push(author_line);
-          });
-          $('#comments-block DD.comment-body', commentsNode).each(function () {
-              var body_line = getTextFromHtml($(this).html()).trim();
-              body.push(body_line);
-          });
-          $('#comments-block DD.comment-footer SPAN.comment-timestamp > A', commentsNode).each(function () {
-              var time_line = $(this).text().trim();
-              time.push(time_line);
-          });
-          //console.log(authors);
-          //console.log(body);
-          //console.log(time);
+        $('#comments-block DT.comment-author', commentsNode).each(function () {
+          var authorText = $(this).text().trim();
+          //the -7 is essentially to remove the 'said...' suffix
+          authorText = authorText.substr(0, authorText.length - 7).trim();
+          authors.push(authorText);
+        });
+        $('#comments-block DD.comment-body', commentsNode).each(function () {
+          var bodyText = getTextFromHtml($(this).html()).trim();
+          body.push(bodyText);
+        });
+        $('#comments-block DD.comment-footer SPAN.comment-timestamp > A', commentsNode).each(function () {
+          var timeText = $(this).text().trim();
+          time.push(timeText);
+        });
       }
-      catch (pex) {
-          //any yawa that gas should not make the original post not to show up right.
+      catch (parseException) {
+        //any yawa that gas should not make the original post not to show up right?
+        console.log(parseException)
```

Comment:
One more thing @dfasoro .  Please replace this with :

``js
console.log("Oops!, Something went wrong. Please try again");
```
We don't want nodejs stacktrace blowing up in users faces :smile: